spec/index.bs

```diff
@@ -1590,6 +1590,10 @@ Pipelines {#pipelines}
 dictionary GPUPipelineDescriptorBase : GPUObjectDescriptorBase {
     required GPUPipelineLayout layout;
 };
+
+interface mixin GPUPipelineBase {
+    GPUBindGroupLayout getBindGroupLayout(unsigned long index);
```

Comment:
should mark with `[SameObject]`

Reply:
I looked at it but the [WebIDL spec](https://heycam.github.io/webidl/#SameObject) says:

> If the [SameObject] extended attribute appears on a read only attribute, then it indicates that when getting the value of the attribute on a given object, the same value must always be returned.

which doesn't seem to apply to methods, so I didn't put it.